app/src/main/java/com/xa/xpensauditor/Day_wise_visualization.java

```diff
@@ -137,33 +141,54 @@ public void onDataChange(@NonNull DataSnapshot datasnapshot) {
                             System.out.println(" Finallyyy   "+amm);
                             int am1=Integer.valueOf(amm);
                             tot=am1+tot;
-      dumm=snapshot.child("Day").getValue().toString();
+                            dumm=snapshot.child("Day").getValue().toString();
+                            dayyy1=Integer.valueOf(dumm);
+                            Calendar c1 = Calendar.getInstance();
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'for' child has incorrect indentation level 28, expected level should be one of the following: 22, 24, 26.